docs/agent-compat-matrix.md

```
@@ -0,0 +1,395 @@
+**ACP adapter:** `@zed-industries/codex-acp` (npm)
+**Config dir:** `~/.codex/` (override: `CODEX_HOME`)
+**Tier:** 2
+
+#### Unrestricted mode
+
+| Mechanism | Details | Per-task? | CLI? | ACP? |
+|-----------|---------|-----------|------|------|
+| CLI flag | `--full-auto` (sandbox=workspace-write + approval=on-request) | Yes | Yes | No |
+| CLI flag | `--yolo` (sandbox=danger-full-access + approval=never) | Yes | Yes | No |
+| CLI flag | `-a never -s danger-full-access` | Yes | Yes | No |
+| `-c` override | `-c approval_policy=never -c sandbox_mode=danger-full-access` | Yes | **Yes** |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)